src/Microsoft.Framework.Runtime/Internal/ConcurrencyUtilities.cs

```diff
@@ -6,9 +6,9 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.Framework.PackageManager
+namespace Microsoft.Framework.Runtime.Internal
 {
-    internal static class ConcurrencyUtilities
+    public static class ConcurrencyUtilities
```

Comment:
Because it is used by the package manager too. Please note that I moved the file in a different project